UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Helpers/FileShareServiceTests.cs

```diff
@@ -213,5 +213,45 @@ public async Task WhenGetBatchInfoBasedOnProducts_ThenReturnsSearchBatchResponse
         }
 
         #endregion GetBatchInfoBasedOnProducts
+
+        #region DownloadBatchFile
+        [Test]
+        public async Task WhenGetBatchInfoBasedOnProducts_ThenDownloadBatchFilesResponse()
+        {
+            A.CallTo(() => fakeAuthTokenProvider.GetManagedIdentityAuthAsync(A<string>.Ignored)).Returns(GetFakeToken());
+            A.CallTo(() => fakeFileShareServiceClient.CallFileShareServiceApi(A<HttpMethod>.Ignored, A<string>.Ignored, A<string>.Ignored, A<string>.Ignored))
+                 .Returns(new HttpResponseMessage() { StatusCode = HttpStatusCode.OK, RequestMessage = new HttpRequestMessage() { 
+                     RequestUri = new Uri("http://test.com") 
+                 }, Content = new StreamContent(new MemoryStream(Encoding.UTF8.GetBytes("Bad request"))) 
+                 });
+
+            var response = await fileShareService.DownloadBatchFiles(new List<string> { "C:\\HOME\\test.txt" }, "C:\\HOME");
```

Comment:
Assuming that "C:\\HOME\\test.txt" is a fake file path it could make it easier to understand if we used a variable to hold the value and name it something like fakeFilePath to make its purpose clearer.

This would be true for other occurrences in these two new tests.

Reply:
Yes. did changes accordingly used variables.